bin/lib/create.js

```diff
@@ -66,8 +66,10 @@ function copyJsAndLibrary (projectPath, shared, projectName, targetAPI) {
     } else {
         fs.ensureDirSync(nestedCordovaLibPath);
         fs.copySync(path.join(ROOT, 'framework', 'AndroidManifest.xml'), path.join(nestedCordovaLibPath, 'AndroidManifest.xml'));
-        TemplateFile.render(path.join(ROOT, 'framework', 'project.properties'), path.join(nestedCordovaLibPath, 'project.properties'), {
-            SDK_VERSION: targetAPI || constants.SDK_VERSION
+        await new Promise((resolve) => {
+            const propertiesEditor = createEditor(path.join(ROOT, 'framework', 'project.properties'));
+            propertiesEditor.set('target', `android-${targetAPI || constants.SDK_VERSION}`);
+            propertiesEditor.save(path.join(nestedCordovaLibPath, 'project.properties'), resolve);
```

Comment:
This code handles no errors. Fortunately, as it turns out, `properties-parser` [supports sync writing](https://github.com/xavi-/node-properties-parser/blob/fb1b7038380fa295ff80ed0d1a1fad3ad1788738/index.js#L367-L371) it is just missing from the documentation. We also already make use of that fact in `GradlePropertiesParser`. So we should probably just use the sync version.

Reply:
Interesting, I can revert that to avoid unnecessary breaking changes then.